pkgs/applications/science/meteorology/supercell-wx/patches/skip-git-versioning.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/scwx-qt/tools/generate_versions.py b/scwx-qt/tools/generate_versions.py
+index e7c470e..dde3e4f 100644
+--- a/scwx-qt/tools/generate_versions.py
++++ b/scwx-qt/tools/generate_versions.py
+@@ -74,15 +74,8 @@ def CollectVersionInfo(args):
+ 
+     versionInfo = VersionInfo()
+ 
+-    repo = git.Repo(args.gitRepo_, search_parent_directories = True)
+-
+-    commitString = str(repo.head.commit)[:10]
+-    
+-    if not repo.is_dirty(submodules = False):
+-        copyrightYear = datetime.datetime.fromtimestamp(repo.head.commit.committed_date).year
+-    else:
+-        commitString  = commitString + "+dirty"
+-        copyrightYear = datetime.date.today().year
++    commitString = "@rev@"
++    copyrightYear = datetime.date.today().year
```

Comment:
Not reproducible. I suppose it doesn't matter much though.

Reply:
I could just hard-code it to the year of the revision being built.

Reply:
I could also modify this script to read it from the LICENSE file. What do you think?

Reply:
If it's not too much effort, then yes.

Reply:
@FliegendeWurst I have updated this patch to read the copyright year from the license file.